app/experimenter/experiments/tests/test_models.py

```diff
@@ -285,14 +285,29 @@ def test_generate_normandy_slug_returns_expected_slug(self):
         experiment = ExperimentFactory.create(
             type=Experiment.TYPE_PREF,
             slug="experiment-slug",
-            firefox_version="57.0",
+            firefox_min_version="57.0",
+            firefox_max_version="59.0",
             firefox_channel="Nightly",
             bugzilla_id="12345",
         )
 
         self.assertEqual(
             experiment.generate_normandy_slug(),
-            "pref-experiment-slug-nightly-57.0-bug-12345",
+            "pref-experiment-slug-nightly-57.0-59.0-bug-12345",
+        )
+
+        experiment = ExperimentFactory.create(
```

Comment:
If this test fails is it clear exactly what's not working?

Reply:
I guess I see what you're saying... that it could be failing due to other parts of the normandy slug formatting... nothing is coming to mind for an improvement though..